src/main/resources/test_files/runtime/FunctionObjectAccumulate.iq

```diff
@@ -0,0 +1,3 @@
+(:JIQS: ShouldRun; Output="( { "b" : [ 2, [ 2, xyz ] ], "c" : [ 3, { "d" : 200.0 } ] } )" :)
```

Comment:
We need tests with keys that do not appear in all objects, as well as with one object, zero objects and three objects.